### DISCUSSION TITLE
Floating windows for sidebar

### DISCUSSION THREAD

**Discussion by AzureRT456** [pulse-browser/browser#240] [Ideas]
Sidebar is nice and definitely one of my main needs in a browser. Although Pulse's sidebar doesn't float but rather pinned, so it's basically just Vivaldi's tab tiling. The ability to have the option for panels to be floating or pinned (for those who want pinned panels) would be nice, as well as ability to separate width for other panels.

**Comment by ethpsantos** [comment]
100% I have no ideia why all Firefox things pinned panels works.
It basically break the webpages because it requires a resize every time you open or close a side panel.

**Reply by AzureRT456** [reply]
I'm afraid I don't understand.

**Reply by ethpsantos** [reply]
I agreed with you.

Firefox have side panels pinned and so when you open or close it will resize the web page displayed on the browser.
For example:
1 - News site opened in the browser
2 - You have a side bar app let's say Whatsapp.

Everytime you have to open the sidepanel of the Whatsapp sidebar app it will resize the News site opened in the browser... that will happen when you close it too.

Pinned side panels break how web sites should be displayed.
So the nedded of floating side panels.

This was one of my issues in the past: https://github.com/pulse-browser/browser/issues/217

**Reply by AzureRT456** [reply]
So I can take it that floating side panels will become an eventual feature. 

Will there be anything regarding this?: https://github.com/pulse-browser/browser/issues/250
